core/App/utils/helpers.ts

```diff
@@ -12,6 +12,34 @@ import { parseUrl } from 'query-string'
 
 import { Attribute, Predicate } from '../types/record'
 
+export function parseCredDefName(credDefId?: string): string {
```

Comment:
I'm thinking this actually might be a good idea to pull the schema and cred def utility functions into their own module as well. They can then be imported and exported directly from this module.

Reply:
change completed